README.md

```diff
@@ -122,6 +122,12 @@ Settings:
 
 6. Visit http://localhost:8000/sdl.php
 
+### Usage with docker
+1. Build locally: `docker build -t goSDL .`
```

Comment:
When running this I got an error `invalid reference format: repository name must be lowercase`
Should we changes this to all lowercase? `docker build -t gosdl .`

Reply:
:+1: